src/reducers/toggleAddCard.js

```diff
@@ -0,0 +1,10 @@
+const toggleAddCard = (state = false, actions) => {
```

Comment:
toggleAddCard suggests it's kind of action you can perform (especially confusing in components)
`this.props.toggleAddCard && <Card >`
Please rename it to something like `isAddCardVisible`

Reply:
ok, renamed